delivery/config.py

```diff
@@ -8,13 +8,15 @@ class Config:
     FLASK_APP = "delivery"
     TESTING = False
     DEBUG = False
+    DB_PATH = None
     PATH_FOR_INITIAL_DATA = None
     HOST = "0.0.0.0"
     PORT = 5000
 
 
 class ProdConfig(Config):
     PATH_FOR_INITIAL_DATA = environ.get("PATH_FOR_INITIAL_DATA")
+    DB_PATH = environ.get("DB_PATH")
```

Comment:
FYI: path is for files, here is it a `connection string` :)

Reply:
fixed